run-it/development/README.md

```diff
@@ -0,0 +1,44 @@
+# Development stack
+
+> ⚠️ **Parabol does not provide any support on this stack**. Use it under your own resposibility.
+
+## General notes
+
+- **This stack is not meant for production use.** It is our development stack and can change at any moment, have errors and incorporate and remove components we are testing without any notice.
+- A .env file localed in the root of this repository is used as [Docker env_file](https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-env_file-attribute). It is the same file used by our command `yarn dev` to run the application in development mode.
+- This stack is designed to be managed using `yarn db:start` and `yarn db:stop` to start the databases and `yarn dev` or `yarn start` to start and manage the application, depending on the use case.
+
+## Components
```

Comment:
IMO we should remove this because the docker-compose file provides all the same information. the best documentation is self-documenting code!

Reply:
I agree that the first sentence could be removed. 

On the second sentence, the docker-compose file does not provide information about the `yarn` commands that can be used to manage the stack. I put that text to be explicit about how should it be used.

Reply:
First sentence was removed. Second sentence is staying for now.